docs/react/quickstart.md

```diff
@@ -354,6 +354,43 @@ return (
 
 Here, when we reload, if there is no app history present, we'll be able to navigate back to our home route.
 
+## Adding Icons
+
+Ionic React comes with (https://ionic.io/ionicons/) pre-installed. All you need to do is import the icon of your choice from the `ionicons` package, and pass it to an `IonIcon` component through the `icon` prop:
+
+```tsx
+import React from 'react';
+import { IonButton, IonContent, IonIcon } from '@ionic/react';
+import { camera } from 'ionicons/icons';
+
+export const IconExample: React.FC = () => {
+  <IonContent>
+    <IonButton>
+      <IonIcon icon={camera} />
+      Take Picture
+    </IonButton>
+  </IonContent>
+};
+```
+
+Note that for React, we're passing the imported SVG reference, NOT the icon name as a string.
```

Comment:
```suggestion
Note that for React, we are passing the imported SVG reference, NOT the icon name as a string.
```

Small thing, but I usually avoid contractions for folks who do not speak English as their primary language.

Reply:
Also for "NOT" can we make this bold rather than all caps?